test/image/mocks/scatter_fill_pattern.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
There is currently a bug in our image test system.
To avoid failures on CircleCI, please rename the new mocks and baselines to start with `z` so that they don't touch the order of rendering baselines: 
`test/image/mocks/scatter_fill_pattern.json` > `test/image/mocks/z-scatter_fill_pattern.json`
`test/image/baselines/scatter_fill_pattern.png` > `test/image/baselines/z-scatter_fill_pattern.png`
Thank you!

Reply:
Sure. renamed in 143f8f1771b9e4f8d34a8390d8460bb36b5e74e0